src/tracking_state.jl

```diff
@@ -63,10 +63,12 @@ struct TrackingState{
         CN <: AbstractCN0Estimator,
         DS <: Union{DownconvertedSignalCPU, Nothing},
         CAR <: Union{CarrierReplicaCPU, Nothing},
-        COR <: Union{Vector{Int8}, Nothing}
+        COR <: Union{Vector{<:Real}, Nothing},
+        PCF <: AbstractPostCorrFilter,
```

Comment:
It is a bit unintuitive, that `PCF` is the last type, while `post_corr_filter::PCF` is the third entry in the struct. I would prefer moving either of them so that the order matches again.